tracim/views/core_api/session_controller.py

```diff
@@ -99,28 +97,36 @@ def bind(self, configurator: Configurator):
         configurator.add_route(
             'login',
             os.path.join(BASE_API_V2, 'sessions', 'login'),
```

Comment:
When debugging APIs, developers search the code for patternes like `/sessions/login`. So do not use os.path.join but string. By the way, if tracim runs on windows, os.path.join will add backslash, which is wrong.

Prefix `/api/v2` should be included using `config.include(routes, route_prefix='/users')``or equivalent, so that your `configure.add_route` become :  `configurator.add_route('login', '/sessions/login')`

See https://github.com/Pylons/pyramid/issues/406 for some examples.